src/Containers/EsiConfiguration.php

```diff
@@ -62,6 +62,11 @@ class EsiConfiguration extends AbstractArrayAccess
         // Redis Cache
         'redis_cache_location' => 'tcp://127.0.0.1',
         'redis_cache_prefix'   => 'eseye:',
+
+        // Memcached Cache
+        'memcached_host' => '127.0.0.1',
+        'memcached_port' => '11211',
+        'memcached_compressed' => true,
```

Comment:
Is there a performance implication with this? I am thinking when a large amount of requests are made, maybe this can negatively affect the cache. More so, keys should 'expire' and therefore automatically clean up to prevent the cache from becoming crazy big. Just a thought.

Reply:
I don't know. Of what I've read, compression is better because on high load, cpu is cheaper than IO/memory.
Also, compression only happened if a specific threshold in saving space if reach, and size of data exceed a specific size. Default is 23% saving and above 2000 bytes.
However, I can set default to false if you prefer.

Reply:
http://php.net/manual/en/memcached.configuration.php